src/app/models/curr-module.ts

```diff
@@ -0,0 +1,40 @@
+import { Curriculum } from './Curriculum';
+import { Module } from './Module';
+
+export class CurrModule {
```

Comment:
Can we refactor this to CurriculumModule?

Reply:
Should be refactored into CurriculumModule where applicable now
- I rename some files and objects to match the new conventions. The only exception is the currModules property on Curriculum since that is what it's called on the backend and I can't change it from whatever the property is called there.